render/tests/test-onremove.js

```diff
@@ -195,13 +195,13 @@ o.spec("onremove", function() {
 			})
 			// Warning: this test is complicated because it's replicating a race condition.
 			o("removes correct nodes when child delays removal, parent removes, then child resolves", function () {
-				// Sugar over the complexity - I need to test the entire tree for consistency.
-				function expect(expectedPairs) {
+				// Custom assertion - we need to test the entire tree for consistency.
+				function template(tpl) {return function (root) {
```

Comment:
If we're going to nest it like that, let's use arrow functions so it's a little more readable.